tests/e2e/cypress/integration/spanish.spec.js

```diff
@@ -0,0 +1,49 @@
+describe("Index page - Internationalization spec", () => {
+  it("Gives user Spanish language option", () => {
+    cy.visit("/")
+
+    cy.get("#header").should("contain", "Español")
+    cy.get("#header").should("not.contain", "English")
+  })
+
+  it("Changes the language to Spanish", () => {
+    cy.visit("/")
+
+    cy.contains("Español")
+      .click()
+
+    cy.contains("La nueva forma de pagar")
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+  })
+
+  it("Changes the language from Spanish back to English", () => {
+    cy.visit("/")
+
+    cy.contains("Español")
+      .click()
+
+    cy.contains("La nueva forma de pagar")
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+    cy.contains("English")
+      .click()
+
+    cy.get("#main-content").should("contain", "The new way to pay for transit")
+    cy.get("#main-content").should("not.contain", "La nueva forma de pagar")
+  })
+
+  it("Changes the language to Spanish from any page", () => {
+    cy.visit("/")
+
+    cy.get(".buttons .btn")
+      .first()
+      .click()
+
+    cy.contains("Español")
+      .click()
+
+    cy.get("#main-content").should("contain", "¡Hagámoslo!")
+    cy.get("#main-content").should("not.contain", "Let’s do it!")
```

Comment:
I think rather than checking for specific content, we should check that the button exists and when clicking it, it changes to the English button while the page URL stays the same. 

That way if we end up inserting new pages / changing the text/order, this test still passes (which is that the button should be on any page).

Reply:
To be resolved in different PR.